LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy-callback-arguments.html

```diff
@@ -47,7 +47,7 @@
       [ "about:blank", "TrustedScriptURL", "HTMLScriptElement src",
           _ => script.src = "about:blank" ],
       [ "2+2", "TrustedScript", "eval", _ => eval("2+2") ],
-      [ "(function anonymous(\n) {\nreturn 2+2\n})", "TrustedScript",
+      [ "function anonymous(\n) {\nreturn 2+2\n}", "TrustedScript",
```

Comment:
Just for the record: this expectation seems to be based on Chromium's behavior, but indeed https://tc39.es/ecma262/#sec-createdynamicfunction does not add parentheses around *sourceString*